index.bs

```diff
@@ -140,43 +140,25 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
   <em>This section is non-normative</em>
 
   <p>
-    This specification relies on exposing the following sets of properties:
-    <ul>
-      <li>
-        <p>
-          An API to query the user agent with regards to the decoding and
-          encoding abilities of the device based on information such as the
-          codecs, profile, resolution, bitrates, etc. The API exposes
-          information such as whether the playback should be smooth and power
-          efficient.
-        </p>
-        <p>
-          The intent of purposes of the decoding capabilities API is to provide
-          a powerful replacement to API such as
-          {{MediaSource/isTypeSupported()}} or
-          {{HTMLMediaElement/canPlayType()}} which are vague and mostly help the
-          callers to know if something can not be decoded but not how well it
-          should perform.
-        </p>
-      </li>
-      <li>
-        <p>
-          Better information about the display properties such as supported
-          color gamut or dynamic range abilities in order to pick the right
-          content for the display and avoid providing HDR content to an SDR
-          display.
-        </p>
-      </li>
-      <li>
-        <p>
-          Real time feedback about the playback so an adaptative streaming can
-          alter the quality of the content based on actual user perceived
-          quality. Such information will allow websites to react to a pick of
-          CPU/GPU usage in real time. It is expected that this will be tackled
-          as part of the [[media-playback-quality]] specification.
-        </p>
-      </li>
-    </ul>
+    This specification defines an API to query the user agent with regards
+    to the audio and video decoding and encoding abilities of the device,
```

Comment:
I think we should be more hand wave about what provides these capabilities... it may be the device, or it could be software on the device. 

```suggestion
    to the audio and video decoding and encoding capabilities,
```